packages/flutter/lib/src/rendering/table_border.dart

```diff
@@ -265,13 +265,13 @@ class TableBorder {
       return true;
     if (runtimeType != other.runtimeType)
       return false;
-    final TableBorder typedOther = other;
-    return top == typedOther.top
-        && right == typedOther.right
-        && bottom == typedOther.bottom
-        && left == typedOther.left
-        && horizontalInside == typedOther.horizontalInside
-        && verticalInside == typedOther.verticalInside;
+    return other is TableBorder
```

Comment:
Same here (as above). We know after the if that other is castable to TableBorder, so we don't really have to check that again, no?

Reply:
Same answer as above.